ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.15__okd-scos.yaml

```diff
@@ -3,7 +3,7 @@ base_images:
     name: "4.15"
     namespace: origin
     tag: base
-  ocp_4.15_installer:
+  origin_4.15_installer:
     name: "4.15"
     namespace: ocp
```

Comment:
@andfasano this should've been reverted to:
```
  okd_scos_4.15_installer:
    name: "4.15"
    namespace: origin
```

Reply:
Thanks for chasing it @LorbusChris , I'll open a new PR to fix it!

Reply:
Addressed in https://github.com/openshift/release/pull/45322